examples/exception_handling/exception_handling.py

```diff
@@ -0,0 +1,90 @@
+import asyncio
+from typing import Callable, Tuple
+
+from golem_core import GolemNode, Payload, commands
+from golem_core.default_logger import DefaultLogger
+from golem_core.default_payment_manager import DefaultPaymentManager
+from golem_core.low.activity import Activity
+from golem_core.low.exceptions import BatchError
+from golem_core.mid import (
+    Buffer,
+    Chain,
+    Limit,
+    Map,
+    default_create_activity,
+    default_create_agreement,
+    default_negotiate,
+)
+
+PAYLOAD = Payload.from_image_hash(
+    "9a3b5d67b0b27746283cb5f287c13eab1beaa12d92a9f536b747c7ae"
+)
+SUBNET = "public"
```

Comment:
I think a better approach could be to just bump the required `yapapi` version. I tested this briefly and nothing unexpected happened.
I don't know about any reason for using `0.9.3` now.

Reply:
I don't think we want to bump yapapi in the middle of removing yappai dependency.

We have this task to handle issue of having this `SUBNET = "public"` in examples
https://github.com/golemfactory/golem-core-python/issues/73